backend/fms_core/serializers.py

```diff
@@ -202,33 +190,41 @@ class SampleExportSerializer(serializers.ModelSerializer):
     location_coord = serializers.CharField(read_only=True, source="container.coordinates")
     location_barcode = serializers.SerializerMethodField()
     current_volume = serializers.SerializerMethodField()
-    projects = serializers.PrimaryKeyRelatedField(read_only=True, many=True)
 
     class Meta:
-        model = Sample
-        fields = ('sample_id','sample_kind', 'sample_name', 'alias', 'cohort', 'taxon',
-                  'container_kind', 'container_name', 'container_barcode', 'location_barcode', 'location_coord',
-                  'individual_id', 'sex', 'pedigree', 'mother_name', 'father_name',
-                  'current_volume', 'concentration', 'collection_site', 'tissue_source', 'creation_date',
-                  'depleted', 'coordinates', 'projects', 'comment' )
+        model = FullSample
+        fields = "__all__"
+        extra_fields = ("taxon", "sex", "pedigree", "cohort", "container_name", "container_kind", "container_barcode")
 
     def get_location_barcode(self, obj):
-        if obj.container.location is None:
+        if obj.container and obj.container.location is None:
```

Comment:
only works because obj.container is always true but otherwise it would be the same as for the mother and father case ...